src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,19 @@
+$days: "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+$days-in-month: 31;
+$cell-size: 100px;
+
+$days-in-week: 7;
+$weekdays: $days-in-week - 1;
+$gap: $cell-size/100;
+
+$max-width: (
+  $cell-size * $days-in-week) + $weekdays * $gap;
+$font-size: round($cell-size/3.33
+);
```

Comment:
I don't understand the formatting

Reply:
Unfortunately is the Linter who is messing the code on save.